app/main/forms/work_order_forms.py

```diff
@@ -0,0 +1,20 @@
+from wtforms import RadioField
+from wtforms.validators import DataRequired
+from dmutils.forms import DmForm
+
+
+class WorkOrderSellerForm(DmForm):
+    seller = RadioField(
+        coerce=int,
+        validators=[
+            DataRequired(message='You must select a seller.')
+        ]
+    )
+
+    def __init__(self, *args, **kwargs):
+        super(WorkOrderSellerForm, self).__init__(*args, **kwargs)
+
+        brief_id = kwargs.pop('brief_id')
+        data_api_client = kwargs.pop('data_api_client')
```

Comment:
You can mix named arguments with the catchall syntax.  E.g.:

`def __init__(self, brief_id, data_api_client, *args, **kwargs)`

This will give you brief_id and data_api_client as arguments for this function, and any other arguments will go in args and kwargs.